### ISSUE TITLE
Kodi, nor NitoTV pop up on my Apple TV 3 jailbroken with Blackb0x. Manual install?

### ISSUE THREAD

**Issue by alectrona6400** [NSSpiral/Blackb0x#129]
Hello, I found a couple of Apple TV 3's at my local Goodwill. My problem right now is that Kodi nor nitoTV are popping up on screen, and I've been searching through open and closed issues on what to do. I've heard of manual installs, nonworking repos... the list goes on.
Also, blackb0x crashes on my MacBook Pro when an Apple TV 3 in normal mode is plugged in. 10.14.6, early2011 patched
Is there a way to manually install nitoTV and Kodi on this Apple TV? Are any other repos still up?
Would like actual solutions please. Thanks in advance.

**Comment by null7elf** [comment]
Just had the same issue - no kodi, no nitoTV

**Comment by MasterTempo567** [comment]
Yup, I've got the same issue combined with after jailbreak I end up in recovery mode and after that it's just stuck until I restart it manually. After it's booted up I don't see anything nitoTV or Kodi related.

**Comment by peanuts007** [comment]
After several unsuccessful tries I decided to select 'restored Apple TV' directly before installing Blackb0x and yes, Kodi appeared after some minutes (tested on version A1469). :-)

**Comment by peanuts007** [comment]
> Yup, I've got the same issue combined with after jailbreak I end up in recovery mode and after that it's just stuck until I restart it manually. After it's booted up I don't see anything nitoTV or Kodi related.

As long as your Apple TV is USB connected to your Mac you will boot into recovery mode. Disconnect USB and power, then reconnect power. For Kodi see my comment above.

**Comment by N1CR0N1X** [comment]
I'm still having trouble trying to get kodi and nito tv to appear, i restored the Apple TV before using blackb0x. any ideas on how to fix the issue?

**Comment by kocoman2** [comment]
does restore cause a forced update to the current version?
